Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -169,6 +169,7 @@ class TextureCacheBase
 
   TCacheEntry* Load(const u32 stage);
   static void InvalidateAllBindPoints() { valid_bind_points.reset(); }
+  static void SetBindPoint(u8 stage, bool value) { valid_bind_points.set(stage, value); }
```

Comment:
Yup, I like that too.  Also you can use `reset` on the bitset instead of set in that case.